05/app.js

```diff
@@ -3,13 +3,56 @@ const zipInputEl = document.querySelector('input[name="zip"]');
 const voivodeshipSelectList = document.querySelector(
 	'select[name="voivodeship"]'
 );
+const inputNameEl = document.querySelector('div input[name="firstName"]');
+const inputSurnameEl = document.querySelector('div input[name="lastName"]');
+const inputStreetEl = document.querySelector('div input[name="street"]');
+const houseNumberInput = document.querySelector('div input[name="houseNumber"]');
+const flatNumberInput = document.querySelector('div input[name="flatNumber"]');
+const cityInputEl = document.querySelector('div input[name="city"]')
 const messageList = document.querySelector(".messages");
 
+
+
+
+function validateCity(){
```

Comment:
Okazuje się, że ta funkcja niż nie zwraca (brakuje `return`) - czyli zwraca `undefined`, a to jest wartość typu `falsy` czyli brakuje `return true` poniżej `displaySuccessMessage('pole poprawnie wypełnione')`

PS. Zobacz komentarz do linii 106 i sprawdź pozostałe funkcje ;)